mbq/metrics/__init__.py

```diff
@@ -22,15 +22,19 @@ def init(namespace=None, constant_tags=None):
         logger.warning('mbq.metrics already initialized. Ignoring re-init.')
         return
 
+    _statsd = create_statsd(namespace=namespace, constant_tags=constant_tags)
+    _initialized = True
+
+
+def create_statsd(namespace=None, constant_tags=None):
```

Comment:
Do you have an example of how you're going to use this `atomiq`? Just want to think through the API.

Reply:
int atomiq's `__init__.py` I would have something like this:
```
from mbq.metrics import create_statsd

_initialized = False
_statsd = None

def init(constant_tags=None):
    global _initialized, _statsd
    if _initialized:
        return

    _statsd = create_statsd(
        namespace='mbq.atomiq',
        constant_tags=constant_tags,
    )
    _initialized = True
```
Then in `settings.py` the user will have to call our `init` function:
```
import mbq.atomiq
mbq.atomiq.init(constant_tags={
    env=ENV_NAME, 
    service='invoicing',
})
```

Then in `mbq.atomiq.consumer` we use the `_statsd` that we initialized.
```
from . import _statsd
metrics_collector = mbq.metrics.Collector(statsd=_statsd)
```